### PR TITLE
Aw moped

### PR DESCRIPTION
**Pull request by gknisely**
initial updates for motor scooter support.  @acwilton and I will tweak the costing as needed.  RAD testing started.  This will not be installed on the data producer in production until RAD testing is complete.

### CODE REVIEW

File: src/valhalla_path_comparison.cc [hunk 1]
```diff
@@ -123,6 +120,7 @@ int main(int argc, char *argv[]) {
   factory.Register("auto", CreateAutoCost);
   factory.Register("bicycle", CreateBicycleCost);
   factory.Register("pedestrian", CreatePedestrianCost);
+  //factory.Register("motor_scooter", CreateMotorScooterCost);
```
Comment (kdiluca) [inline src/valhalla_path_comparison.cc]: Is this suppose to be commented?

**Comment by kdiluca** [comment]
:+1:

File: src/valhalla_path_comparison.cc (hunk 1 above)
Comment (gknisely) [reply src/valhalla_path_comparison.cc]: thanks fixed

**Comment by gknisely** [comment]
All rad tests passed.

**Pull request merged**